content/pl/concepts/_index.md

```diff
@@ -0,0 +1,70 @@
+
+Gdy tylko zdefiniujesz zamierzony stan, warstwa sterowania Kubernetes (*Kubernetes Control Plane*) podejmuje działania, aby aktualny stan klastra był zgodny ze stanem zamierzonym, wykorzystując do tego Pod Lifecycle Event Generator ([PLEG](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/pod-lifecycle-event-generator.md)). W tym celu Kubernetes wykonuje szereg automatycznych zadań, takich jak start lub restart kontenerów, skalowanie liczby replik dla danej aplikacji itp. Warstwa sterowania Kubernetes to zbiór różnych procesów działających na klastrze:
+
+* **Kubernetes Master** to zbiór trzech procesów uruchamianych na pojedynczym węźle klastra, który pełni rolę węzła _master_. Te procesy to: [kube-apiserver](/docs/admin/kube-apiserver/), [kube-controller-manager](/docs/admin/kube-controller-manager/) oraz [kube-scheduler](/docs/admin/kube-scheduler/).
+* Na każdym węźle klastra nie będącym węzłem typu _Master_ działają dwa procesy:
+  * **[kubelet](/docs/admin/kubelet/)**, który komunikuje się z Kubernetes Master.
+  * **[kube-proxy](/docs/admin/kube-proxy/)**, proxy sieciowe pośredniczące w usługach sieciowych Kubernetes.
+
+## Obiekty Kubernetes
+
+Kubernetes składa się z różnych abstrakcyjnych obiektów, które reprezentują stan systemu: wdrożone aplikacje i zadania w kontenerach, powiązane zasoby sieciowe i dyskowe oraz inne informacje o tym, co się dzieje na klasterze. Te abstrakcyjne obiekty są reprezentowane przez API Kubernetes. [Understanding Kubernetes Objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/) zawiera więcej szczegółów na ten temat.
+
+Do podstawowych obiektów Kubernetes należą:
+
+* [Pod](/docs/concepts/workloads/pods/pod-overview/)
+* [Service *(Serwis)*](/docs/concepts/services-networking/service/)
+* [Volume *(Wolumin)*](/docs/concepts/storage/volumes/)
+* [Namespace](/docs/concepts/overview/working-with-objects/namespaces/)
+
+Kubernetes zawiera także obiekty abstrakcyjne wyższego poziomu, zbudowane z obiektów podstawowych przy wykorzystaniu [Kontrolerów](/docs/concepts/architecture/controller/), które dostarczają dodatkowe funkcjonalności i udogodnienia. Należą do nich:
+
+ * [Deployment](/docs/concepts/workloads/controllers/deployment/)
+ * [DaemonSet](/docs/concepts/workloads/controllers/daemonset/)
+ * [StatefulSet](/docs/concepts/workloads/controllers/statefulset/)
+ * [ReplicaSet](/docs/concepts/workloads/controllers/replicaset/)
+ * [Job](/docs/concepts/workloads/controllers/jobs-run-to-completion/)
+
+## Warstwa sterowania (*Kubernetes Control Plane*)
```

Comment:
I suggest adding an explicit fragment identifier:
```suggestion
## Warstwa sterowania (*Kubernetes Control Plane*) {#warstwa-sterowania)
```

Does that make sense? It's a nice-to-have, totally optional tweak.

Reply:
It'd be nice to have, I agree, but why to put it just here? And why there are no such identifiers in the upstream?

Reply:
Upstream only has one language. Not vital to make that change.
The advantage? It means stability even if the localization convention changes, or if the original English text changes and the localized version does not. Tiny detail, right?

Reply:
Good idea, I think we should keep it also in other places, especially if special chars are included (special chars should be removed, then).